app/models/ticket.js

```diff
@@ -44,5 +44,13 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  taxIncurred: computed('event.tax', function() {
+    let taxType = this.event.get('tax.isTaxIncludedInPrice');
```

Comment:
Use ES6 getters.

Reply:
@pradeepgangwar actually `tax` is a promise object here, that's why I used `event.get('tax.isTaxIncludedInPrice')`, to resolve the promise

Reply:
@uds5501 If you are explicitly calling for tax in the model, why are you using event.tax, just use tax? Since you already fetched it there?

Reply:
@uds5501 I agree with @CosmicCoder96 . Resolve the tax promise at the model level and use directly in code here.